modules/ROOT/pages/oauth-dance-about.adoc

```diff
@@ -3,37 +3,42 @@ ifndef::env-site,env-github[]
+== OAuth Dance Authentication Flow
 
-. A token is requested to the provider, specifying client application credentials as needed (the credentials correspond to a Client ID and secret generated by API Platform).
-. Mule OAuth 2.0 provider returns a token.
-. An HTTP request is performed against the protected resource (ie resource where the OAuth 2.0 Access Token Enforcement policy is applied), including the token either as an authentication header or a query parameter.
-- The protected resource is linked with the API definition through Autodiscovery.
-. The OAuth 2.0 Access Token Enforcement policy intercepts this request and communicates with the Provider to validate the token.
+The OAuth Dance authentication flow is illustrated in the following screenshot:
+image::mule-oauth-provider-dance.png[oauth+policy1,height=366,width=640]
+
+. A token request is sent to the provider, specifying client application credentials as needed. The credentials correspond to a client ID and secret generated by API Platform.
+. The Mule OAuth 2.0 provider returns a token.
+. An HTTP request is performed against the protected resource to which the OAuth 2.0 Access Token Enforcement policy is applied.
++
+The protected resource *must* be linked with the API definition through Autodiscovery.
+. The OAuth 2.0 Access Token Enforcement policy intercepts this request and communicates with the provider to validate the token.
 . If the token is correctly validated:
-- The token is whitelisted and kept on record until expiration. Any further requests that contain this token are not validated against the OAuth provider.
-- The associated client ID of the client application credentials is checked to make sure that the application has an active contract with the API. (Not Applicable to OpenAM)
-. The request is forwarded to the resource.
+.. The token is whitelisted and kept on the record until it expires. Any further request that contains this token is not validated against the OAuth provider.
+.. The associated client ID of the client application credentials is verified to ensure that the application has an active contract with the API. This is not applicable to OpenAM providers.
+. The request is then forwarded to the resource.
 . An HTTP response is generated.
 
-The requester gets authorization from an OAuth provider instead of directly gaining access to the credentials of the client application. The client application owner, owns the credentials and authorizes the provider to interact with the API. The protected resource knows the user is authorized because it has a valid access token.
+The requester obtains the authorization from an OAuth provider, instead of directly gaining access to the credentials of the client application. The client application owner, owns the credentials and authorizes the provider to interact with the API. The protected resource determines that the user is authorized because it has a valid access token.
```

Comment:
no comma
```suggestion
The requester obtains the authorization from an OAuth provider, instead of directly gaining access to the credentials of the client application. The client application owner owns the credentials and authorizes the provider to interact with the API. The protected resource determines that the user is authorized because it has a valid access token.
```

Reply:
Done! Thanks!